test/models/Career/Model.test.ts

```
@@ -60,13 +60,33 @@ describe("Career model", () => {
     });
   });
 
+  it("raise an error if credits is 0", async () => {
```

idem: `should raise an error if credits is 0`